content/sensu-go/6.3/operations/deploy-sensu/generate-certificates.md

```diff
@@ -296,6 +296,84 @@ TODO: Document steps for adding CA root to Windows trust store
 
 {{< /language-toggle >}}
 
+## Renew self-generated certificates
+
+Certificates expire, so you'll need to renew them from time to time to keep your Sensu deployment running smoothly.
+Depending on how your certificates are configured, one backend certificate may expire before the others, all three backend certificates may expire at the same time, or the agent certificate may expire before any of the backend certificates.
+
+This section explains how to find certificate expiration dates, confirm whether certificates have already expired, and renew certificates.
+
+### Find certificate expiration dates
+
+Use this check to find certificate expiration dates.
+This allows you to renew certificates before they expire to avoid observability interruptions.
+
+**TODO: Add check example**
+
+command: openssl x509 -noout -enddate -in cert.pem
+
+### Confirm expired certificates
+
+Use `sensuctl cluster health` to check your deployment for expired backend certificates.
+
+The following `sensuctl cluster health` response indicates that one backend certificate is expired:
+
+{{< code shell >}}
+Error: GET "/health": Get https://localhost:8080/health?timeout=3: x509: certificate has expired or is not yet valid
+{{< /code >}}
+
+The log for the expired backend will be similar to this example:
+
+{{< code shell >}}
+backend-01_1      | {"component":"etcd","level":"warning","msg":"health check for peer 4cc76e198efb22e8 could not connect: remote error: tls: bad certificate (prober \"ROUND_TRIPPER_RAFT_MESSAGE\")","pkg":"rafthttp","time":"2021-06-22T20:40:54Z"}
+backend-01_1      | {"component":"etcd","level":"warning","msg":"health check for peer 8495a7b2d2ee8597 could not connect: remote error: tls: bad certificate (prober \"ROUND_TRIPPER_RAFT_MESSAGE\")","pkg":"rafthttp","time":"2021-06-22T20:40:54Z"}
+{{< /code >}}
+
+If you restart the cluster with one expired backend certificate, the `sensuctl cluster health`response will include an error:
+
+{{< code shell >}}
+Error: GET "/health": failed to request new refresh token; client returned 'Post https://localhost:8080/auth/token: EOF'
+{{< /code >}}
+
+The following `sensuctl cluster health` response indicates that all three backend certificates are expired:
```

Comment:
this can also happen for other reasons but if you get the other messages then this is just further confirmation